l10n.toml

```diff
@@ -103,5 +103,69 @@ branches = [
 [env]
 
 [[paths]]
-  reference = "components/**/src/main/res/values/strings.xml"
-  l10n = "components/**/src/main/res/values-{android_locale}/strings.xml"
+  reference = "components/browser/engine-system/src/main/res/values/strings.xml"
```

Comment:
@Pike the bot is bringing the changes back, it looks like it doesn't have the latest version of AC.

Reply:
Yes, I know.

There's a technical problem I'm working on right now that prevents us from taking updates from a-c into android-l10n.

I was hoping we'd win the race today.

My suggestions is to close this PR for today.

Reply:
Thanks!